### PR TITLE
sync with Plugin skeleton v1.1.0-RC

### PR DESCRIPTION
**Pull request by lsmith77**
see https://github.com/Sylius/PluginSkeleton/compare/6a47794e594a7bbdb335aa737355459e8868d86d...master

### CODE REVIEW

File: src/Importer/ImporterResult.php [hunk 1]
```diff
@@ -88,10 +88,10 @@ public function getFailedRows(): array
...
 
     /**
-     * @return int The duration (in milliseconds)
+     * @return float The duration (in milliseconds)
      */
-    public function getDuration(): int
+    public function getDuration(): float
     {
-        return (int) $this->stopWatchEvent->getDuration();
+        return $this->stopWatchEvent->getDuration();
```
Comment (lsmith77) [inline src/Importer/ImporterResult.php]: https://github.com/symfony/symfony/issues/26204

File: .gitignore [hunk 2]
```diff
@@ -7,3 +7,5 @@
 
 /etc/build/*
 !/etc/build/.gitkeep
+
+/tests/Application/yarn.lock
```
Comment (solutionDrive-Alt) [inline .gitignore]: maybe no big deal in .gitignore, but the blank line at the end of the file is missing

File: composer.json [hunk 3]
```diff
@@ -16,8 +16,6 @@
     "require-dev": {
         "portphp/excel": "dev-master",
```
Comment (solutionDrive-Alt) [inline composer.json]: side-question: do we still need dev-master here?

Comment (lsmith77) [reply composer.json]: yes https://github.com/portphp/excel/compare/1.0.0...master :(

File: .gitignore (hunk 2 above)
Comment (lsmith77) [reply .gitignore]: fixed

**Pull request merged**